aries-site/src/pages/templates/forms/managing-child-objects.mdx

```diff
@@ -0,0 +1,74 @@
+import { Example } from '../../../layouts';
+
+## When to use
```

Comment:
This is great content and very informal however some of this to me sounds more `why to use` rather then `when to use`  

for example:
This pattern improves productivity by focusing a user's attention solely on the current object of interest, and uses collapsible objects to show or hide detail as needed. The intent is that collapsible objects reduce noise, clutter, and the resulting cognitive overload by hiding extraneous detail — and an easy means to view detail when needed.

I would reduce this with:
Use this pattern when a user needs to expand/collapse on different child objects as an easy means to view detail when needed. (can work on wording) 

```

This pattern improves productivity by focusing a user's attention solely on the current object of interest, and uses collapsible objects to show or hide detail as needed. The intent is that collapsible objects reduce noise, clutter, and the resulting cognitive overload by hiding extraneous detail 

```

This is more informed about the component pattern itself rather than `when to use`
just my opinion, unless we want this in this section just thinking out loud

Reply:
Good comments. I tend to agree. I originally had they "why" portion above as part of the purpose statement, but it didn't flow as well. I don't want to delete it outright either, as I think we need our content to educate readers as to the rationale/why so that they have the deeper level of understanding.

I'll take another pass.

Reply:
yeah totally agree it is great information and I wouldn't want to take it out. I also thought about a `why` section but its hard since some of our pages don't have similar flow I was thinking in terms of how the `layer` page is getting done but did not see a good section in there. Happy to brainstorm as well if depending on second pass

Reply:
@britt6612 - I reworked this a bit. See if it flows better for you now.